tests/fixtures/sample_data.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+import pytest
+import xarray as xr
+
+
+@pytest.fixture
+def one_individual():
```

Comment:
Good job tidying up the fixtures to a separate file.

However, the convention we've followed so far is that the file holding the fixtures is named (roughly) after the sub-module being tested. Naming this file `sample_data.py` may mislead some to thinking that it contains fixtures intended for testing the `sample_data.py` sub-module.

Since these fixtures are (currently) only used for testing the plots, I would simply name this file `plots.py`.